features/332-sso/api.md

```diff
@@ -80,6 +78,48 @@ skygear.auth.getLoginRedirectResult().then(user => {
 });
 ```
 
+## React Native
+
+### API
+
+```typescript
+interface SSOLoginOptions {
+  onUserDuplicate?: "abort" | "merge" | "create";
+}
+
+function loginOAuthProvider(providerID: string, callbackURL: string, options?: SSOLoginOptions): Promise<User>;
+function linkOAuthProvider(providerID: string, callbackURL: string): Promise<User>;
+
+function loginApple(providerID: string, callbackURL: string, options?: SSOLoginOptions): Promise<User>;
+function linkApple(providerID: string, callbackURL: string): Promise<User>;
+```
+
+### Example
+
+```typescript
+function LoginScreen() {
+  // For a React Native app, Sign in with Apple must provide two experiences.
+  //
+  // On iOS, the developer must use loginApple which uses AuthenticationServices
```

Comment:
It seems there are more cases here:
- iOS >= 13: use AS
- Android / iOS < 13: use standard OIDC flow

Is it possible to unify these three cases into single call to `loginApple`?

Reply:
The provider ID is different. (`apple-app` vs `apple-web`). I expect the developer to use [react-native-device-info](https://github.com/react-native-community/react-native-device-info) to check the version themselves.

Reply:
Is there any reason that the provider IDs have to be different? Would there be problems if developer would want to link the principals of the same Apple ID signed up using `apple-app` & `apple-web` together?

Reply:
The opaque user ID we obtain from Apple is scoped under the team. So as long as the team remains the same, different App IDs and different Services IDs still get the same opaque user ID.

Reply:
I have updated the example to use react-native-device-info